reana_commons/workflow_engine.py

```diff
@@ -66,6 +66,8 @@ def load_cwl_operational_options(ctx, param, value):
     cwl=dict(load_operational_options_callback=load_cwl_operational_options),
     serial=dict(load_operational_options_callback=load_json),
     yadage=dict(load_operational_options_callback=load_yadage_operational_options),
+    # TODO: Create `load_snakemake_operational_options`.
```

Comment:
`s/TODO/FIXME/g` since I believe some of Python code checkers warn about FIXMEs, but not TODOs... But perhaps that changed in the meantime.